pages/linux/systemctl.md

```diff
@@ -11,26 +11,26 @@
 
 `systemctl --failed`
 
-- Start/Stop/Restart/Reload a service:
+- Start/Stop/Restart/Reload/Show the status a service:
 
-`systemctl {{start|stop|restart|reload}} {{unit}}`
-
-- Show the status of a unit:
-
-`systemctl status {{unit}}`
+`systemctl {{start|stop|restart|reload|status}} {{unit}}`
 
 - Enable/Disable a unit to be started on bootup:
 
 `systemctl {{enable|disable}} {{unit}}`
 
-- Mask/Unmask a unit to prevent enablement and manual activation:
+- Reload systemd, scan for new or changed units:
+
+`systemctl daemon-reload`
 
-`systemctl {{mask|unmask}} {{unit}}`
+- Check if a unit is active/enabled/failed:
 
-- Reload systemd, scanning for new or changed units:
+`systemctl {{is-active|is-enabled|is-failed|}} {{unit}}`
 
-`systemctl daemon-reload`
+- List all units of service/socket/automount filter by running/failed state:
+
+`systemctl list-units --type={{service|socket|automount}} --state={{failed|running}}`
 
-- Check if a unit is enabled:
+- Show the source unit files:
 
-`systemctl is-enabled {{unit}}`
+`systemctl cat {{unit}}`
```

Comment:
Technically, most `systemctl` subcommands accept a pattern or unit as their positional argument. So if we change this one, we should probably change everything else too. It will also be a good idea also to define what exactly is a "pattern" in this context.

What I'm saying is I think it's too complex to be worth being "technically accurate", so I'm in favour of leaving it as `{{unit}}`. More advanced users can always refer to the manual.

Reply:
So "that the given pattern" is not enough, right?

Yeah, I think it doesn't help users who don't know at leadt the basic of regular expressions.